config/factions/neutral.json

```diff
@@ -2,7 +2,6 @@
 	"neutral" :
 	{
 		"name" : "Neutral",
-		"index" : 9,
```

Comment:
Looks to be largely unused. RMG uses it to define zones without towns. No uses other than that.
Should probably add special constant for that RMG case, e.g. NONE=-2 so it won't get allocated to any mod.

Interesting detail - there is nothing that would prevent modders from giving town to this "neutral" faction. So they can break RMG already.

Reply:
"there is nothing that would prevent modders from giving town to this "neutral" faction" - yes, but it will be mod problem, not engine problem. Another story is when mod is good but game is broken

Reply:
Decided to revert this change - RMG case is more complex than I thought and don't want to play "find all regressions before release" game.

Instead, changed warning rules in regards to "index" field - now, it can only be used by "core" mod. This resolves original reason behind this change